pptx_blueprint/__init__.py

```diff
@@ -54,7 +54,27 @@ def _find_shapes(self, label: str) -> Iterable[BaseShape]:
         Args:
             label (str): label of the placeholder (without curly braces)
         """
-        pass
+        slide_number, tag_name = label.split(":")
```

Comment:
Maybe I missed this decision, but did we really decide for this 'slide_number:label' notation? And why? This is an internal method and we thus have the data under control. Considering a use in the replace_text method, we have the label as variable as well as a separate scope variable which would be essentially the list/set of slide numbers. In particular, if you have a list of slide numbers, how would this 'colon notation' look like? Please refactor to use two variables :)